src/components/Dashboard/UserMenu.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useContext, useState, Fragment } from 'react';
+import { makeStyles } from '@material-ui/styles';
+import Avatar from '@material-ui/core/Avatar';
+import Button from '@material-ui/core/Button';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+import IconButton from '@material-ui/core/IconButton';
+import LogoutVariantIcon from 'mdi-react/LogoutVariantIcon';
+import AuthContext from '../../utils/AuthContext';
+
+const useStyles = makeStyles(theme => ({
+  avatar: {
+    height: theme.spacing(6),
+    width: theme.spacing(6),
+    padding: 0,
+  },
+}));
+
+export default function UserMenu() {
```

Comment:
We added a "Show Access Token" button to our existing dropdown recently, to make things a bit easier for some folks who run some scripts on their laptop. Can we do similar here (or add some other easy way to copy an access token).

Reply:
Good idea. I filed https://github.com/mozilla-frontend-infra/balrog-ui/issues/4.